### ISSUE TITLE
Incorrect type for the body body

### ISSUE THREAD

**Issue by eriktews** [mozilla/OpenWPM#354]
* **I'm submitting a ...**
[X] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

* **Summary**

The current type used for the body_body is String as shown here:

https://github.com/mozilla/openwpm-webext-instrumentation/blob/778da7aeff9a0f60a6516e9a94481e1f305f2e9f/src/schema.ts#L70

However in the current implementation, the post body is a dict since Firefox returns the post body as a dictionary with the parsed form data. Since TypeScript seems to check the types only during compile time but not during runtime and the dict is convert to any first and then assigned, this is not noticed. OpenWPM later converts that to JSON, so that still works well.

So in a nutshell, the code here isn't really clean, but it works anyway (but it might break later).

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
